apps/passport-server/test/util/env.ts

```diff
@@ -30,10 +30,10 @@ export const zuzaluTestingEnv: EnvironmentVariables = Object.freeze({
 export const pcdpassTestingEnv: EnvironmentVariables = Object.freeze({
   ...zuzaluTestingEnv,
   IS_ZUZALU: "false",
-  PRETIX_ORG_URL: undefined,
-  PRETIX_TOKEN: undefined,
-  PRETIX_VISITOR_EVENT_ID: undefined,
-  PRETIX_ZU_EVENT_ID: undefined,
+  PRETIX_ORG_URL: "pretix_org_url",
+  PRETIX_TOKEN: "pretix_token",
+  PRETIX_VISITOR_EVENT_ID: "visitor_event_id",
+  PRETIX_ZU_EVENT_ID: "zu_event_id",
```

Comment:
Since PCDPass needs to use the Pretix service now, these values are required.

Using the Pretix service is the simplest way of populating the DB with Zuzalu user records, but it might be overkill if we're not expecting the data to ever change in Pretix. In that case, we could just do a one-off import of users via a SQL batch job.